src/components/AboutApp.js

```diff
@@ -23,6 +22,7 @@ export const AboutApp = () => {
           <img src='/images/wikipedia-wordmark-en.png' />
         </div>
         <div class='version'>
+          {/* eslint-disable-next-line no-undef */}
           <p>{APP_VERSION}</p>
```

Comment:
Now would be a good time to make that app version available through a utils and have that eslint suppression in one place only.